exercises/D-strings/exercise.js

```diff
@@ -1,3 +1,15 @@
 // Start by creating a variable `message`
 
-console.log(message);
+//Write a program that logs a message and its type
+const number = 4;
+let message = "Rahwa";
+let text;
```

Comment:
We use `const` if the content of the variable will not change, and if it changes we use `let`. So in this case all of them should have been `const`.